### ISSUE TITLE
Create 2020 landing page and launch CFP 

### ISSUE THREAD

**Issue by irabinovitch** [socallinuxexpo/scale-drupal#36]
We've selected dates and venue for 2020 and are hoping to get a landing page and initial CFP launched.  

SCALE 18x will be held March 5-8, 2020 at the Pasadena Convention Center.
We want to link to https://register.socallinuxexpo.org/reg6/ for registration

@ScislaC offered to help with designs if we need it.

**Comment by irabinovitch** [comment]
@rgolan do you have availability to help again this year?

**Comment by rgolan** [comment]
Hi Ilan,

I don’t think I can devote the time that the SCALE site really deserves. I think the website hasn’t been up to the level it should be. I’d be happy to help get someone else familiar with what we now have or helping someone else that wants to take on this task.

—
Ron Golan
rgolan@superconcentrated.com

**Comment by irabinovitch** [comment]
Thanks @rgolan . Appreciate all your help over the years.  Hope we'll still see you during scale.

**Comment by irabinovitch** [comment]
@btmash let me know if you need any more info to get the landing page going.

**Comment by btmash** [comment]
@toddzebert I agree but I'm mostly trying to get the lay of the land.

@irabinovitch Thanks for that. I created the taxonomy term `18x`. I then created the various pages that go with all of this, updated the slogan on the site settings page. Created the 18x header and footer menus and copied last year's blog post with the updated dates.

We can see the prelim site at https://www.socallinuxexpo.org/scale/18x (the menus are not in place - they will come with the theme).

So this now leaves us with 2 remaining things:

- [ ] Graphics for 2020 (SCaLE 18x logo + banner)
- [ ] Theme for this year

I'll get the theme up and running soon. But the current PR needs to be merged in as well, @irabinovitch as its a blocker for further work.
